### PR TITLE
Feature/unique check errors

### PR DESCRIPTION
**Pull request by Geist5000**
changes to support changes of backend : https://github.com/ricom/toolbox-backend/pull/10

### CODE REVIEW

**Comment by Geist5000** [comment]
[Nur registrieren mit Jade-HS mail](https://trello.com/c/bBsykIzB/310-nur-registrieren-mit-jade-hs-mail)

File: src/general-components/UniqueCheck/UniqueCheck.tsx [hunk 1]
```diff
@@ -140,12 +130,64 @@ export class UniqueCheck extends Component<ReplaceProps<"input", FormControlProp
...
+                        this.setState({
+                            success: call.callData.data.available,
+                            reason: call.callData.data.reason,
+                            isLoading: false,
+                            error: false
+                        });
+                        return;
+                    }
+                } catch (reason) {
+                    console.dir(reason);
```
Comment (nic-schi) [inline src/general-components/UniqueCheck/UniqueCheck.tsx:175]: remove this

**Review by nic-schi — CHANGES_REQUESTED** [review]
Looks very good, theres one console.dir that can be removed tho

File: src/general-components/UniqueCheck/UniqueCheck.tsx (hunk 1 above)
Comment (Geist5000) [reply src/general-components/UniqueCheck/UniqueCheck.tsx]: added check if react is in debug mode and only prints if it is

**Review by ma1160 — APPROVED** [review]

**Review by nic-schi — APPROVED** [review]
Approved.

**Comment by Geist5000** [comment]
[Registrieren von bereits vorhandenen E-Mails - bessere Rückmeldung](https://trello.com/c/nqZNelfD/434-registrieren-von-bereits-vorhandenen-e-mails-bessere-r%C3%BCckmeldung)

**Pull request merged**